src/main/drivers/rx/rx_sx1280.c

```diff
@@ -338,58 +342,141 @@ uint8_t sx1280GetStatus(void)
+        PreambleLength = 8;
+    PreambleLength = ((PreambleLength / 4) - 1) << 4;
+
+    uint8_t buf[7];
+    buf[0] = PreambleLength;                    // AGCPreambleLength
+    buf[1] = SX1280_FLRC_SYNC_WORD_LEN_P32S;    // SyncWordLength
+    buf[2] = SX1280_FLRC_RX_MATCH_SYNC_WORD_1;  // SyncWordMatch
+    buf[3] = HeaderType;                        // PacketType
+    buf[4] = PayloadLength;                     // PayloadLength
+    buf[5] = SX1280_FLRC_CRC_3_BYTE;            // CrcLength
+    buf[6] = 0x08;                              // Must be whitening disabled
+    sx1280WriteCommandBurst(SX1280_RADIO_SET_PACKETPARAMS, buf, 7);
+
+    // CRC seed (use dedicated cipher)
+    buf[0] = (uint8_t)(crcSeed >> 8);
+    buf[1] = (uint8_t)crcSeed;
+    sx1280WriteRegisterBurst(SX1280_REG_FLRC_CRC_SEED, buf, 2);
+
+    // Set SyncWord1
+    buf[0] = (uint8_t)(syncWord >> 24);
+    buf[1] = (uint8_t)(syncWord >> 16);
+    buf[2] = (uint8_t)(syncWord >> 8);
+    buf[3] = (uint8_t)syncWord;
+
+    // DS_SX1280-1_V3.2.pdf - 16.4 FLRC Modem: Increased PER in FLRC Packets with Synch Word
+    if (((cr == SX1280_FLRC_CR_1_2) || (cr == SX1280_FLRC_CR_3_4)) &&
+        ((buf[0] == 0x8C && buf[1] == 0x38) || (buf[0] == 0x63 && buf[1] == 0x0E)))
+    {
+        uint8_t temp = buf[0];
+        buf[0] = buf[1];
+        buf[1] = temp;
+        // For SX1280_FLRC_CR_3_4 the datasheet also says
+        // "In addition to this the two LSB values XX XX must not be in the range 0x0000 to 0x3EFF"
+        if (cr == SX1280_FLRC_CR_3_4 && buf[3] <= 0x3e)
+            buf[3] |= 0x80; // 0x80 or 0x40 would work
+    }
+
+    sx1280WriteRegisterBurst(SX1280_REG_FLRC_SYNC_WORD, buf, 4);
+
+    // Set permissible sync errors = 0
+    sx1280WriteRegister(SX1280_REG_FLRC_SYNC_ADDR_CTRL, sx1280ReadRegister(SX1280_REG_FLRC_SYNC_ADDR_CTRL) & 0xf0);
+}
+
+void sx1280Config(const uint8_t bw, const uint8_t sfbt, const uint8_t cr,
+                  const uint32_t freq, const uint8_t preambleLength, const bool iqInverted,
+                  const uint32_t flrcSyncWord, const uint16_t flrcCrcSeed, const bool isFlrc)
+{
+    sx1280SetMode(SX1280_MODE_STDBY_RC);
+
+    sx1280PacketMode = (isFlrc) ? SX1280_PACKET_TYPE_FLRC : SX1280_PACKET_TYPE_LORA;
+    sx1280WriteCommand(SX1280_RADIO_SET_PACKETTYPE, sx1280PacketMode);
+
+    if (isFlrc) {
+        sx1280ConfigModParamsFlrc(bw, cr, sfbt);
+        sx1280SetPacketParamsFlrc(preambleLength, SX1280_FLRC_PACKET_FIXED_LENGTH, 8, flrcSyncWord, flrcCrcSeed, cr);
+    } else {
+        sx1280ConfigModParamsLora(bw, sfbt, cr);
+        sx1280SetPacketParamsLora(preambleLength, SX1280_LORA_PACKET_FIXED_LENGTH, 8, SX1280_LORA_CRC_OFF, iqInverted);
+    }
+
+    sx1280SetDioIrqParams(
+      SX1280_IRQ_TX_DONE | SX1280_IRQ_RX_DONE | SX1280_IRQ_SYNCWORD_VALID | SX1280_IRQ_SYNCWORD_ERROR | SX1280_IRQ_CRC_ERROR, // irqMask
+      SX1280_IRQ_TX_DONE | SX1280_IRQ_RX_DONE, // dio1Mask
+      SX1280_IRQ_RADIO_NONE,
+      SX1280_IRQ_RADIO_NONE
```

Comment:
Indentation should be 4 spaces.
```suggestion
        SX1280_IRQ_TX_DONE | SX1280_IRQ_RX_DONE | SX1280_IRQ_SYNCWORD_VALID | SX1280_IRQ_SYNCWORD_ERROR | SX1280_IRQ_CRC_ERROR, // irqMask
        SX1280_IRQ_TX_DONE | SX1280_IRQ_RX_DONE, // dio1Mask
        SX1280_IRQ_RADIO_NONE,
        SX1280_IRQ_RADIO_NONE
```